pkg/kubectl/cmd/logs.go

```diff
@@ -224,6 +225,11 @@ func (o LogsOptions) Validate() error {
 		return errs.ToAggregate()
 	}
 
+	var err error
+	o.GetPodTimeout, err = cmdutil.GetPodRunningTimoutFlag(cmd)
+	if err != nil {
```

Comment:
Since this is the last thing you do you don't need this check, just `return err`.